app/scripts/metamask-controller.js

```diff
@@ -395,6 +400,17 @@ const PHISHING_SAFELIST = 'metamask-phishing-safelist';
 // OneKey devices can connect to Metamask using Trezor USB transport. They use a specific device minor version (99) to differentiate between genuine Trezor and OneKey devices.
 export const ONE_KEY_VIA_TREZOR_MINOR_VERSION = 99;
 
+const environmentMappingForRemoteFeatureFlag = {
+  [ENVIRONMENT.DEVELOPMENT]: 'dev',
```

Comment:
Nit: We could be using the enums from `@metamask/remote-feature-flag-controller` here instead